src/python-common/ceph/tests/test_drive_group.py

```diff
@@ -334,3 +463,119 @@ def test_raw_ceph_volume_command_1():
     sel = drive_selection.DriveSelection(spec, inventory)
     with pytest.raises(ValueError):
         cmds = translate.to_ceph_volume(sel, []).run()
+
+@pytest.mark.parametrize("test_input5",
+[
+    (
+        """service_type: osd
+service_id: testing_drivegroup
+placement:
+  host_pattern: hostname
+method: raw
+data_devices:
+  paths:
+  - path: /dev/sda
+    crush_device_class: hdd
+  - path: /dev/sdb
+    crush_device_class: hdd
+  - path: /dev/sdc
+    crush_device_class: hdd
+db_devices:
+  paths:
+  - /dev/sdd
+  - /dev/sde
+  - /dev/sdf
+
+"""
+    ),
+    ])
+def test_raw_ceph_volume_command_2(test_input5):
+
+    spec = DriveGroupSpec.from_json(yaml.safe_load(test_input5))
+    spec.validate()
+    drive = drive_selection.DriveSelection(spec, spec.data_devices.paths)
+    cmds = translate.to_ceph_volume(drive, []).run()
+
+    assert (cmds[0] == 'raw prepare --bluestore --data /dev/sda --block.db /dev/sdd', 'raw prepare --bluestore --data /dev/sdb --block.db /dev/sde', 'raw prepare --bluestore --data /dev/sdc --block.db /dev/sdf') # noqa E501
+
+@pytest.mark.parametrize("test_input6",
+[
+    (
+        """service_type: osd
+service_id: testing_drivegroup
+placement:
+  host_pattern: hostname
+method: raw
+data_devices:
+  paths:
+  - path: /dev/sda
+    crush_device_class: hdd
+  - path: /dev/sdb
+    crush_device_class: hdd
+  - path: /dev/sdc
+    crush_device_class: ssd
+db_devices:
+  paths:
+  - /dev/sdd
+  - /dev/sde
+  - /dev/sdf
+
+"""
+    ),
+    ])
+def test_raw_ceph_volume_command_3(test_input6):
+
+    spec = DriveGroupSpec.from_json(yaml.safe_load(test_input6))
+    spec.validate()
+    drive = drive_selection.DriveSelection(spec, spec.data_devices.paths)
+    cmds = translate.to_ceph_volume(drive, []).run()
+
+    assert (cmds[0] == 'raw prepare --bluestore --data /dev/sda --block.db \
+            /dev/sdd --crush-device-class hdd', 'raw prepare --bluestore \
+            --data /dev/sdb --block.db /dev/sde --crush-device-class hdd',
+            'raw prepare --bluestore --data /dev/sdc --block.db /dev/sdf \
+                    --crush-device-class ssd') # noqa E501
+
+@pytest.mark.parametrize("test_input7",
```

Comment:
@adk3798 this should be the required test with 3 raw paths, 3 db_devices and 3 wal_devices, with two different device classes

Reply:
tests look good, thanks